Automattic/jetpack
# JS Components: Fix multiline visual issue in Alert component

To fix this visual issue, this PR wrappers the icon with a div element, and tweaks/updates the styles on it.

Fixes #

#### Changes proposed in this Pull Request:

* JS Components: Fix multiline visual issue in Alert component

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Use storybook
* Go to Alert story: http://localhost:50240/?path=/docs/js-packages-components-alert--default
* Play with the Alert component: add long content to create multi-lines in the message

before | after
------|------
<img width="613" alt="Screen Shot 2022-06-20 at 1 43 41 PM" src="https://user-images.githubusercontent.com/77539/174594643-792a724f-5fb4-4c95-a307-a91e0424152f.png"> |  <img width="611" alt="image" src="https://user-images.githubusercontent.com/77539/174987928-c93c0b63-7baf-44ee-81e4-825fa5ad7fac.png">




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202469108953672
